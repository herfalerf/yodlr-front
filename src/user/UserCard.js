import React from "react";
import "./UserCard.css";

const UserCard = ({ id, firstName, lastName, email, state }) => {
  return (
    <div className="User-Card card">
      <div className="card-body">
        <h6 className="card-title">
          {firstName} {lastName}
        </h6>
        <p>
          <small>User Id: {id}</small>
        </p>
        <p>
          <small>Email: {email}</small>
        </p>
        <p>
          <small>Active: {state}</small>
        </p>
      </div>
    </div>
  );
};

export default UserCard;
