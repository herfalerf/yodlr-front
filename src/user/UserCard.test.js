import React from "react";
import { render } from "@testing-library/react";
import UserCard from "./UserCard";
import { MemoryRouter } from "react-router";

it("matches snapshot", function () {
  const { asFragment } = render(
    <MemoryRouter>
      <UserCard
        id="17"
        firstName="Test"
        lastName="Testuser"
        email="testemail@test.com"
        state="active"
        key="17"
      />
    </MemoryRouter>
  );
  expect(asFragment()).toMatchSnapshot();
});
