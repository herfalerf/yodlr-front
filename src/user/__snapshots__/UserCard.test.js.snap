// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="User-Card card"
  >
    <div
      class="card-body"
    >
      <h6
        class="card-title"
      >
        Test Testuser
      </h6>
      <p>
        <small>
          User Id: 17
        </small>
      </p>
      <p>
        <small>
          Email: testemail@test.com
        </small>
      </p>
      <p>
        <small>
          Active: active
        </small>
      </p>
    </div>
  </div>
</DocumentFragment>
`;
